synadm/cli/user.py

```diff
@@ -138,6 +139,62 @@ def deactivate(ctx, helper, user_id, gdpr_erase):
         click.echo("Abort.")
 
 
+@user.command()
+@click.argument("regex", type=str)
+@click.option(
+    "--gdpr-erase", "-e", is_flag=True, default=False,
+    help="""Marks the users as GDPR-erased. This means messages sent by the
+    users will still be visible by anyone that was in the room when these
+    messages were sent, but hidden from other users joining the room
+    afterwards.""", show_default=True
+)
+@click.option(
+    "--dry-run", "-n", is_flag=True, default=False,
+    help="""Do everything except deactivating users."""
+)
+@click.option(
+    "--batch-size", "--paginate", "-p", type=int, default=500,
+    show_default=True,
+    help="""How many users should be requested from the API one at a time.
+    This option has no effect on how many users will be deactivated.
+
+    Increasing this is not necessary in most cases but useful if you have a
+    lot of accounts on your homeserver."""
+)
+@click.pass_obj
+@click.pass_context
+def deactivate_regex(ctx, helper, regex, gdpr_erase, dry_run, batch_size):
```

Comment:
After re-adding batch_size, the forgotten renaming of this option is not required anymore since I readded `--batch-size` in the first position which makes it the variable name `batch_size`

Reply:
?

Reply:
You changed the option name in your last commit to paginate. That wouldn't have been enough. It wouldn't have worked.

Reply:
No problem, just saying that it wouldn't have worked but now all good :-) No worries!!

Reply:
> You changed the option name in your last commit to `--paginate` but forgot to change the last argument to `paginate`.

ah, understood.